pallets/gear/src/tests.rs

```diff
@@ -1285,7 +1285,7 @@ fn uninitialized_program_should_accept_replies() {
             WASM_BINARY.to_vec(),
             vec![],
             Vec::new(),
-            99_000_000u64,
+            90_000_000u64,
```

Comment:
What's the reason of changing this?

Reply:
Because, when we run smart contracts for block we look at gas limit of message. If it's too big for current block then we will skip its handling. So, I remove hard checks in allocator and smart contracts eat less gas, so gas limit is increased for next message handling and it's skipped in test run to next block.

Reply:
To avoid this skip - I need to reduce gas limit for this message.